### ISSUE TITLE
Deprecation warning in import

### ISSUE THREAD

**Issue by Nolent** [Azure-Samples/cognitive-services-speech-sdk#870]
**Describe the bug**
Importing speech to text raise a DeprecationWarning

**To Reproduce**
```
from warnings import filterwarnings
filterwarnings('error')

from azure.cognitiveservices.speech import (
    CancellationDetails,
    CancellationErrorCode,
    Connection,
    SpeechConfig,
    SpeechRecognitionCanceledEventArgs,
    SpeechRecognitionEventArgs,
    SpeechRecognizer,
)
```

**Expected behavior**
Should not launch DeprecationWarning

**Version of the Cognitive Services Speech SDK**
The problem is here since 1.14.0 . It works perfectly in 1.13.0

**Platform, Operating System, and Programming Language**
 - OS: Debian buster
 - Hardware: x64
 - Programming language: Python

**Additional context**
```
Traceback (most recent call last):
  File "/test.py", line 7, in <module>
    from azure.cognitiveservices.speech import (
  File "/usr/local/lib/python3.7/dist-packages/azure/cognitiveservices/speech/__init__.py", line 8, in <module>
    from .speech import *
  File "/usr/local/lib/python3.7/dist-packages/azure/cognitiveservices/speech/speech.py", line 8, in <module>
    from . import speech_py_impl as impl
  File "/usr/local/lib/python3.7/dist-packages/azure/cognitiveservices/speech/speech_py_impl.py", line 28, in <module>
    _speech_py_impl = swig_import_helper()
  File "/usr/local/lib/python3.7/dist-packages/azure/cognitiveservices/speech/speech_py_impl.py", line 15, in swig_import_helper
    import imp
  File "/usr/lib/python3.7/imp.py", line 33, in <module>
    DeprecationWarning, stacklevel=2)
DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses```

**Comment by pankopon** [comment]
Thanks for reporting this. It seems Python wheels for Linux in the 1.14.0 release were accidentally built using SWIG 3.0.8 which generates deprecated code, instead of SWIG 3.0.12 that was used previously. (The used SWIG version can be seen in the header comments of speech_py_impl.py. The code block that triggers the warning, `from sys import version_info` ... `del version_info` right in the beginning, was generated differently in 1.14.0 than in 1.13.0.)

I've added a work item about this to our sprint backlog and it should be fixed by the 1.15.0 release which is due early next year. We'll update this issue when the actual time of release with the fix can be confirmed.

**Comment by pankopon** [comment]
Just to update, we made a fix for this and it will be included in the Speech SDK 1.15.0 release. The release is currently expected to be out before the end of January next year.
